packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/app/initialization/app_setup.dart

```diff
@@ -31,14 +31,17 @@ Future<void> setupAndRunApp(
 /// such as Firebase or background handlers can be configured here.
 Future configureApp(EnvironmentConfig envConfig) async {
   // TODO: Use flutterfire_cli to create the firebase_options.dart file
-  //  containing the actual configuration values.
-  //  https://firebase.google.com/docs/flutter/setup
+  // containing the actual configuration values.
+  // https://firebase.google.com/docs/flutter/setup
+  //
+  // Below is a minimal configuration of the firebase config to make the project
+  // work out of the box. Please replace it with your own values.
   await safeRun(() => Firebase.initializeApp(
         options: const FirebaseOptions(
-          apiKey: 'apiKey',
-          appId: 'appId',
-          messagingSenderId: 'messagingSenderId',
-          projectId: 'projectId',
+          apiKey: 'AAaaAaAAaa0aa0AA_aAAA0a0a0a0AaOaAaA0aAA',
```

Comment:
Please make sure this is not a real api key and also add a TODO in the code to communicate that this config needs to be adjusted

Reply:
This has been documented and is not a real key. The values are replaced with the appropriate symbols (capital letters, lowercase letters,numbers,underscore) to match the expected patterns.